docs/security-o2k.md

```diff
@@ -0,0 +1,119 @@
+# OpenAPI2Kong support for security directives
+
+This document documents the use of `security` directives within the `deck` subcommand `openapi2kong`. There are 2 sections;
+
+1. [OpenAPI 3 security directives explained](#openapi-3-security-directives-explained)
+2. [Conversion logic to Kong plugins](#conversion-logic-to-kong-plugins)
+
+---
+
+# Openapi 3 security directives explained
+
+Within OpenAPI the security directives can be specified on the document root (the `OpenAPI` object). It can also be specified on the `Operation` object, in which case it will override the document level one.
+Specifically; it cannot be specified on the `path` object level, the level in between document and operation.
+
+## Format
+
+The `security` property is an array of `Security Requirement` objects.
+
+## Behaviour
+
+- The requirements listed are a logical `OR`; passing any one requirement listed is enough to be allowed access.
```

Comment:
This does not describe the full ability as it relates to OAS and your examples below.   Both `OR` and `AND` configurations are possible, as you describe lower in this doc.

Reply:
clarified, this only describes the `security` directive (which handles `OR`). The `securityRequirement` object defines the `AND`.